src/integrationTest/java/uk/gov/hmcts/darts/task/service/AutomatedTaskServiceTest.java

```diff
@@ -96,10 +95,10 @@ class AutomatedTaskServiceTest extends IntegrationPerClassBase {
     private UnstructuredAudioDeleterProcessor unstructuredAudioDeleterProcessor;
 
     @Autowired
-    private UnstructuredToArmProcessor unstructuredToArmProcessor;
+    private AutomatedTaskProcessorFactory unstructuredToArmProcessor;
 
     @Autowired
-    private ArmResponseFilesProcessor armResponseFilesProcessor;
+    private AutomatedTaskProcessorFactory processorFactory;
```

Comment:
maybe rename this to indicate it refers to arm response files? if thats the case :-)

Reply:
Updated name to armResponseFilesProcessor